dir.c

```diff
@@ -3448,6 +3448,83 @@ rb_dir_s_empty_p(VALUE obj, VALUE dirname)
  *  (<code>config.h</code> and <code>main.rb</code>), the parent
  *  directory (<code>..</code>), and the directory itself
  *  (<code>.</code>).
+ *
+ *  == What's Here
+ *
+ *  \Class \Dir provides methods that are useful for:
+ *
+ *  - {Creating}[#class-Dir-label-Creating]
+ *  - {Reading}[#class-Dir-label-Reading]
+ *  - {Setting}[#class-Dir-label-Setting]
+ *  - {Querying}[#class-Dir-label-Querying]
+ *  - {Iterating}[#class-Dir-label-Iterating]
+ *  - {Other}[#class-Dir-label-Other]
+ *
+ *  === Creating
```

Comment:
As `new` doesn't create a directory, there would only be one entry in this section.  I would move both of these methods to the Other section.